docs/examples/modalalert/doClearCommunicate.js

```diff
@@ -1,33 +1,48 @@
 // @flow strict
-import { type Node } from 'react';
-import { ModalAlert, CompositeZIndex, FixedZIndex, Layer, Text } from 'gestalt';
-
-const HEADER_ZINDEX = new FixedZIndex(10);
-const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
+import { Fragment, useState, type Node } from 'react';
+import { Box, Button, ModalAlert, CompositeZIndex, FixedZIndex, Layer, Text } from 'gestalt';
 
 export default function DoClearCommunicate(): Node {
+  const [showComponent, setShowComponent] = useState(false);
+
+  const HEADER_ZINDEX = new FixedZIndex(10);
```

Comment:
Not sure why these were moved inside the components in the examples — consts like these that don't depend on component props should be defined outside the function component so they're not re-created on every render